File: backend/Backend.Core/DTOs/User/SupplierSettingsDto.cs
﻿namespace Backend.Core.DTOs.User;

public class SupplierSettingsDto
{
    public int Id { get; set; }
    public string PublicName { get; set; } = null!;
    public string ApiName { get; set; } = null!;
    public string? Description { get; set; } = null!;
    public bool Searchable { get; set; }
    public string YmlLoadType { get; set; } = null!;
    public string? YmlLink { get; set; } = null!;
    public int? RefreshTimeId { get; set; }
}

File: backend-old/src/account/supplier/dto/supplier-settings.dto.ts
import { WebsiteLink } from '@app/models'
import { YmlCatalogLink } from '../../../../libs/models/src/user/model-inners/yml-link'

export class SupplierSettingsDto {
	publicName?: string
	apiName?: string
	ymlLoadType: 'link' | 'file'
	ymlLinkSettings?: YmlCatalogLink
	description: string
	links: Array<WebsiteLink>
}
